R/validate-landings.R

```diff
@@ -35,12 +48,69 @@ validate_landings <- function(log_threshold = logger::DEBUG){
              deployed_imeis) %>%
     purrr::map_dfr(tibble::as_tibble)
 
-  # CREATE VALIDATED OUTPUT -----------------------------------------------
+  logger::log_info("Validating surveys trips...")
+  surveys_time_alerts <- validate_surveys_time(
+    data = landings,
+    hrs = pars$validation$landings$survey_time$max_duration %||% default_max_limit,
+    submission_delay = pars$validation$landings$survey_time$submission_delay)
+
+  logger::log_info("Validating catches values...")
+  surveys_price_alerts <- validate_catch_price(
+    data = landings,
+    method = pars$validation$landings$prices$method %||% default_method,
```

Comment:
I did not found `prices` into the config file

Reply:
Good catch! I didn't put it in the config file as it was not different from the default values but still put it in the code so that we can change that in the future if needed. Do you think it would be clearer if we put the prices in the config file?